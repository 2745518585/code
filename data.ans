17.5848421209
43.9731754233
24.1415610640
2.2223139451
49.8980094307
5.8857993625
8.5941725105
12.5284566539
14.0798065165
2.8434510488
5.9581267765
29.9575125897
4.1892839101
17.0083585667
46.3409492520
36.3810630188
23.4263765296
53.2520662236
8.3766287181
8.9069537394
4.6830276231
9.7995207301
17.2068580932
1.5730323434
13.4966862688
22.8092881729
2.3466798905
7.6201270264
11.4858808990
66.8088749920
15.1766152438
8.9755234585
81.1755419464
12.3063772857
16.0851619355
3.6621539735
5.6365476008
22.2349556440
27.0827827043
4.6199148380
3.5953232751
20.5690672003
15.6177890953
46.5079341859
8.4031863098
4.8391321617
14.7810042286
7.9209161719
48.0169795163
5.4071342546
4.8939925885
45.2652469945
16.2839285066
47.2529164229
19.4590052431
32.7531659531
35.6270244152
18.1406776217
18.9087080399
20.2232035956
2.8679871787
17.9596957115
2.9586881331
5.1711351909
26.3059757775
11.2605951492
8.5824240787
21.3796077701
38.9360928753
41.6820894930
17.3423365065
6.2605570143
14.5616700102
58.7813104163
2.6950742859
29.4897769728
4.8865888558
2.5891721349
13.5091090898
0.6227205789
28.3316141409
17.2745356020
3.6490352913
29.7988726089
2.3106211909
10.9153112884
4.5973350931
59.5216140990
15.2183752502
2.4674935602
30.4584086797
26.3071909426
1.6892869312
0.5393860533
27.1856632511
3.2011274093
1.8861258459
15.5641085000
39.6584169424
5.4694921116
2.2769590966
2.9157657871
42.2817588417
8.9122662325
27.4436218994
5.8027975309
4.0280037473
2.0608648328
69.8969233321
2.4485436792
14.4249073271
4.1451165729
15.3149669549
3.9273383121
27.7476608098
2.2820008988
25.0447065717
54.0506001974
0.3144022501
8.5069216095
3.2944454708
3.1582313808
31.6573358371
18.5411716821
5.8839631955
4.0020754638
20.8899862826
0.9234168572
2.1644434891
3.7281483940
26.7359758614
3.8793034876
1.8104297714
21.0722433677
39.9738003636
10.1474782301
44.8110181622
10.9924753491
7.5966549210
15.7045150361
16.8071203690
6.3442644311
11.1004063630
9.7986963022
2.9138770422
0.9562710750
10.4737377327
26.9913013697
42.5015302263
7.6021961374
0.2723471496
8.2502401233
19.3500080390
32.1040934680
11.4632543647
31.7760180445
40.8501278807
38.1485573938
18.9571116446
34.4846609615
16.6175172847
2.9638282790
6.5165644479
32.5798304371
21.5820886847
18.1167221681
17.8912560250
5.3110092563
18.2731190455
13.6163535504
8.8048116465
3.1643151990
86.6009793224
1.5463341678
13.9579875529
31.2982880346
10.6183186538
33.6730227605
7.1734048209
36.0837292745
35.4304567016
18.4125399833
13.3945658679
7.7308770936
17.2758482369
8.6471474230
42.5817253095
34.1318246945
42.9896920399
5.9706309715
18.3220374433
0.6359438030
11.6833721489
73.3267749765
47.7012838577
20.0100845770
1.1678273677
5.0686066782
17.3028089411
7.8959386022
40.1984369793
26.5082225378
9.2628838591
13.1214062841
59.9562221615
29.9977481515
1.0383910229
17.5872701925
34.3805157065
18.3622830947
6.6731733768
26.5503863388
9.9737722122
13.3312305154
107.6659564134
23.5555586088
6.7723970907
13.2307592799
54.6253846563
27.4075519665
8.2383487503
1.1863359433
16.1112786426
28.1347858095
0.5191140937
4.0410218519
31.3291503433
36.7459189674
42.7428192439
1.8273937456
19.8139426321
12.8517515568
4.9364763099
17.9471458945
34.0744805240
31.7632689241
9.3253153121
22.0016578078
9.9449661430
7.0252453254
5.9546636264
24.2560518085
55.4200399896
33.2506039936
6.3370728828
59.0232287023
7.0076112908
44.7151847232
45.8563769691
77.1567950419
1.3809995817
19.5421688787
36.4538854825
23.8189652517
2.9115791678
17.8971326542
2.2011787069
30.7826940699
15.9537225171
43.2508318524
3.5038952190
31.3308787357
12.2770972346
45.5711369835
4.3980262850
13.8780239707
9.4772405885
37.7527534580
2.9984252168
3.6629019038
4.7775240802
10.8612777230
6.0274026379
15.4432280686
44.7756499811
0.9215603271
3.2676646915
0.3698539247
33.7150826419
49.4307055501
4.8834992594
8.8750594690
11.6784955477
4.0167417743
19.0937622193
0.2505062370
13.2949957857
1.6515674308
7.4917431102
9.4256524409
11.1981442971
5.4931477126
2.6371012822
31.2041966506
0.9826062592
5.8781491602
5.0851093793
11.3850792890
13.3355391618
26.1733186853
7.7382711654
51.9348890001
8.9987075876
23.3928880041
42.4303029221
1.3081637969
4.7203157467
10.6872117916
5.7279884682
19.1263329900
22.1095512182
48.8958331847
25.8868134542
8.7801271650
17.9918836243
15.5834050432
4.0887876500
12.3071738218
25.2534519278
8.9777972188
35.2353414168
44.7141713572
38.2711853796
20.1588321630
15.9716902286
4.3139077247
9.2364438576
61.8582099198
10.2557828488
35.2268805403
25.8938218512
30.5797541074
12.3923409527
19.0106781533
3.6153029210
14.9742423847
0.2068723295
0.2342329319
12.3412068620
20.1755091126
57.0361609510
28.3456389029
21.5332177184
32.2739085614
14.7636859258
32.8890154083
4.5155506277
0.8603991534
8.2545270504
14.6947427012
10.8219196708
8.9683240298
3.4079546263
1.3132769066
13.6415000520
2.4463973743
18.5821740202
18.4129182828
27.5249009881
21.3487969978
3.3198941767
41.1626735987
17.2212845780
2.8029452198
11.6144891383
31.0565929913
6.6771344674
1.7533915462
26.3305256954
10.6761898947
26.9751426122
3.3788935423
8.4495807070
4.7611684047
9.3817516458
28.0415612525
5.0554956136
13.5194189897
37.5671944670
59.5214676749
22.1682138291
22.6388976454
16.7297194209
1.5179781239
7.3620654053
4.9001373595
1.7272013264
4.3011424897
9.0638016148
16.5302425919
42.8654717653
3.8147907137
5.3823558395
15.2719119864
3.6962679449
48.2757340751
6.3089246405
1.5800663571
29.0428701368
1.4683046046
12.4636473035
18.8082660364
5.8001225437
11.1630772985
20.6121906558
7.2281554581
13.8676300623
3.9094703277
10.2495794976
3.3810427611
37.6060008527
0.9627531546
19.1019472439
12.9057750357
19.9034302475
12.9211889002
16.8555474169
7.9945957695
36.2873887495
47.2507402040
38.4733250881
0.5037905586
5.9659337044
9.5498565442
21.9003140441
30.2170460909
25.0059489709
53.6242982574
7.5658705784
2.6405155668
17.1379967206
0.3515547440
118.5072462174
9.8615736346
61.0000357320
22.1793695818
34.9197739349
25.2495820288
11.8156695153
26.3028067459
20.8494172834
2.2769915183
2.0028895151
1.1419353921
10.6882272351
22.1312167082
56.7359637001
1.4322638256
8.2815238268
7.1576243865
17.4580083780
26.6637453055
0.0462665323
18.6484289969
20.6951262973
33.8770508721
23.9163150327
2.5842484417
31.2200051430
25.6748588567
8.5704160321
2.3463695713
14.1550535396
5.7339439020
9.9041368525
16.9082001797
2.4581717373
23.5912045529
59.5743953834
23.3206696886
40.2419328563
9.0362079377
22.9993846161
11.7732280032
43.2825202309
4.3132348398
39.1533359820
9.1743998354
39.9767043063
0.4248428630
37.3770650627
21.5291991828
6.0051800959
14.9140782601
76.5716879378
36.3177333445
63.7166041637
9.5487249504
7.2875298114
46.5960920174
8.2496652124
17.6822734717
18.3979552669
4.9831672553
14.8247673138
10.6083267633
1.1370226215
12.3265390900
29.9939173671
44.9164974758
0.2395210220
41.0099053186
2.5113061518
2.5976282103
6.7796337291
1.8852445393
2.4499899364
44.6542652967
24.2126386509
16.1504408673
3.5633457213
33.0522232291
9.2423448083
35.4838142670
3.0704436034
2.2520666426
14.3743463888
27.1655640612
19.8024064952
3.5338956010
14.1305433172
13.5222688042
19.5245299256
31.8775021264
23.2088564545
3.7419035885
25.9603562773
1.3765737196
8.2078096494
4.5510474435
0.7102379942
13.6056525115
43.6350386134
61.9676766915
11.6831121705
13.0672489189
26.8731963816
5.5318031068
6.6827584739
29.8105501293
23.1926016455
128.0193501442
15.8158107351
44.8086278581
10.4364628747
15.0656534813
3.0741941588
4.1536810420
21.9678452116
8.4529201249
18.5103213901
2.3627784969
32.8543534333
6.2606112508
19.9721042122
11.8170751096
11.9841598547
0.9520319202
11.1080569677
6.1550110711
25.2788288470
6.3812990715
6.9029737799
21.9788499718
34.7346943959
4.7070980922
19.3330637499
32.4737098165
1.4533151822
26.0677283539
10.2162491175
29.6931254003
6.2416184416
1.0885994865
1.4348123391
33.2325286716
5.7171731527
15.7440236438
9.9315915483
6.6320948748
4.4095119901
4.7047628655
78.8660937661
2.3671086632
4.2838945435
37.3494902881
18.4897460890
6.7651755995
6.6353164367
11.9988122225
28.4220163713
25.2146677346
24.9240580345
15.1714389340
13.9545336666
5.1915584193
9.0341672998
9.4491603500
23.0275556786
18.1546002551
12.3433239617
13.8671700525
1.7733033307
8.5782168912
31.7644570638
48.1417356649
28.2537279316
74.7755170975
9.3035695930
51.9003622120
7.5259733968
24.7144892303
8.5695841596
7.4077023172
47.4670976689
31.2563147308
8.3629420871
31.5524068710
35.6298865436
25.4568338808
7.5232331409
7.0421040973
37.5623099764
4.8356291181
37.6378671770
10.6387663587
43.4571720927
24.6462748108
24.8283482145
35.6411317175
21.8781115099
9.3174579010
30.9904705005
20.9438660907
20.5172508896
0.6603488826
0.8111006155
3.8714579946
22.6140762856
7.4511543828
5.2168681389
10.1224496964
22.9625973328
24.3012034219
4.0737736724
14.0720309832
16.3395714305
37.0011348428
52.7404322644
38.4804374069
4.2761095845
6.8361338172
27.9442287942
11.5934074834
8.7009365793
31.1939685356
52.0932845063
7.8082516054
16.1468250962
9.6265215722
30.8076627728
9.2877360772
7.0980778544
2.4488086557
16.5449231271
22.9648620794
64.5347642020
43.5752043933
3.9140658411
9.4289997351
8.4761479036
26.8560796411
32.2913016636
9.6207211588
89.9150549242
48.5092071470
18.1344293541
4.4056663173
23.8596302703
16.4350966187
30.0507717906
11.5256642232
0.2746576252
25.3652591579
16.4429948498
28.8919783476
11.8604001950
27.8462672101
4.0547362524
41.5741287850
19.9818854652
26.9750827764
47.5326522632
2.1376109937
17.1002405537
17.1928952972
0.1020220283
10.9645285608
48.9547089086
5.0612169456
37.8221296712
17.9770381922
8.0882228681
2.8247687617
19.7014045793
27.5377509502
51.6744407256
15.5587052286
9.8140590785
11.0013697419
12.3132670202
4.3607360579
4.7868700576
36.6148919605
8.6703053979
26.8585159714
38.3251220027
2.0542069677
4.0007481313
5.4750788719
18.4047051552
9.8918523429
34.0738883217
3.3996788243
7.1095358715
19.7925020456
2.0846657713
28.9117678407
28.9985942698
7.1783606604
27.0933598964
16.2572665189
102.0918384073
0.2091664788
0.8384166866
7.9204050664
8.0141914291
13.6626272285
10.7480568105
36.3890618010
12.3600859862
6.5516791133
30.9914511772
68.2439146571
9.8509939438
37.1345127356
96.5249758249
10.8067312487
22.0699927006
23.2199862240
40.0158536542
5.7991385465
0.0831805365
20.4915554996
11.2203167904
0.3953451654
4.7471473836
6.8168914727
44.6003112042
18.1078391078
49.9765158047
4.5038833868
7.9015824998
19.7648605844
3.6078311801
10.0539515487
42.4838263098
16.5597581630
24.8254560829
1.4957784311
22.9727132431
7.8478785773
15.4751007773
4.6413333359
29.0518044871
1.8015141769
6.2561759349
12.0088722057
4.7402238190
7.0559519417
63.6385557168
10.0737649207
19.4922179094
4.3362428952
43.3041020284
54.8957831585
45.3978434423
24.4961135468
74.2893443000
23.5029160733
6.2845444509
13.9936775328
29.1839916690
13.8711899003
7.4163908533
2.7831248990
6.5447851825
8.8685059692
24.9633516349
3.3194052648
10.9237900511
39.9833146103
26.5921326826
5.3277524446
0.6753073235
13.4831518144
3.0140061981
6.1280829380
28.5949930524
10.7933324565
20.9845502197
20.1772595651
24.4755172436
28.4991099047
14.0933438597
9.8731449395
0.9867060528
10.6733142125
6.8119454536
16.4376626104
27.1007724651
20.3990025441
40.5142832913
18.0267958341
1.0061279424
39.4318442367
23.4791316025
6.4438584877
15.5706705143
11.8985582236
4.1336047693
13.3690398053
0.9258876544
1.8469967903
21.2563783201
24.4141906429
19.7536150982
34.2964368535
6.3100760601
41.4692899973
61.3226532605
10.5353448676
3.0584608594
5.9203114699
26.9159201318
41.4733700865
7.8872831376
46.6235837172
28.1630869107
35.3492404814
0.8815647909
10.7147878088
17.2860366524
2.6968451805
7.2961300404
50.3945986837
6.2270134352
16.7299010646
18.9213081549
113.1164776807
35.6699506551
33.7239322420
17.8779494182
12.3810210596
0.9374843562
53.1589672007
0.4603993141
10.0964512044
8.4813879255
12.6975064689
13.6583628606
6.9918111272
31.9200361561
35.9376887628
18.2977220521
23.7437712447
54.2907824420
7.2904933465
10.6211209682
53.3186982645
2.0019034801
11.8659602982
0.4556580061
19.7380763445
4.2219901000
8.5853095583
4.3400336313
56.9989326020
35.1868407681
0.8350541817
24.1835885758
3.1532048960
13.0783581646
25.9120007697
40.3320160667
18.6463566136
36.1453366736
30.7283331912
5.8973049670
5.4193272296
2.6883206062
10.5259237158
3.8102583338
15.6454268769
9.3713521198
25.6259720550
0.3665629419
21.2367725579
23.6468541980
59.8980054368
1.3629289599
6.8044981785
17.2863172897
5.3749222269
2.2347583372
4.1421300039
4.6909810610
37.5760508207
12.3009994032
13.8643905399
6.1594887784
10.4009072147
20.8523102345
6.1770326258
24.4537304381
15.9137924287
0.8802260908
26.9634856991
28.3400998116
16.0838664589
5.9170667587
5.9310681622
7.8266198144
39.6837420640
26.8668687826
3.6213021127
1.0290225589
0.6741433202
17.6559744432
15.3984916505
9.7855025023
3.4869054408
1.1433278003
35.8308955409
2.9620651951
4.5097530410
14.1384600860
30.4817094834
26.2796289415
4.4408479688
17.5022063213
17.0001489111
2.4230471583
8.6946822700
26.0991993948
14.9700631844
13.8779695005
2.1882405811
6.1137956116
12.3007118096
10.7239170618
12.3967132270
4.6708072969
3.7472466530
18.0491829554
18.4524274730
11.4168381265
2.2821589474
6.9929701496
51.9031187246
49.9256322814
16.9189487223
1.5151191561
2.6551835629
1.3637830528
31.5457320130
8.3318835152
4.4401646469
14.7837273394
10.8795477635
2.2924231109
0.0208487843
13.0563695038
24.4849496660
3.3684365612
6.1103676505
31.8238539689
20.5593832883
18.9049320854
14.6487628335
27.2299449436
