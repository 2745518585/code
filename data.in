3 
1 8 5 10 8 
1 6 7 2 6 
0 2 