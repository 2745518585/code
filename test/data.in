1
10 10
2 8 5 4 8 4 1 5 10 10
6 7 2 6 3 4 1 1 3 5
