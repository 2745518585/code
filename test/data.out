Yes
1 13
13 1
1 16
16 20
16 19
13 17
17 15
19 14
15 10
14 9
10 8
9 7
8 6
7 5
6 4
5 3
4 2
20 11
16 12
3 18
